src/main/java/com/andersen/HiberanteDemo.java

```diff
@@ -0,0 +1,76 @@
+package com.andersen;
+
+import com.andersen.bus.TicketType;
+import com.andersen.dao.UserTicketDaoImpl;
+import com.andersen.entity.Client;
+import com.andersen.entity.Ticket;
+import lombok.extern.slf4j.Slf4j;
+
+import java.time.LocalDateTime;
+
+@Slf4j
+public class HiberanteDemo {
+
+	private static final UserTicketDaoImpl USER_TICKET_DAO = new UserTicketDaoImpl();
```

Comment:
try use UserTicketDaoImpl  USER_TICKET_DAO = new UserTicketDaoImpl();

Reply:
do mean declare this line inside main body?

Reply:
No, the one on the left uses abstraction and the one on the right uses implementation.